### PR TITLE
Point to C13's Task List

### PR DESCRIPTION
**Pull request by kaidamasaki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+
+### Wait, I'm in Cohort 13 how did I get here then?
+
+That's our bad, we forgot to update the calendar link from last time to point at this Cohort's fork of task-list.  Sorry about that!
+
+### Oh no, I forked this already!
+
+Don't panic!  You will need to [delete your old C12 fork](https://help.github.com/en/github/administering-a-repository/deleting-a-repository) and create a new C13 fork.  If you haven't made any commits already then you're good to go after doing this.
+
+#### Umm... what if I already made some commits?
```
Comment (jmaddox19) [inline README.md:20]: I've tried to walk a student through this process before and ended up realizing 15 minutes in that I don't even know how to do this. Could we either (1) change this to just say "it's ok, for this project instructors will look for PRs in the C12 fork as well" or (2) writeup a process for _how_ to do this?

**Review by jmaddox19 — COMMENTED** [review]
Haha. I love the humor! Thanks for doing this! Just one concern.

File: README.md (hunk 1 above)
Comment (kaidamasaki) [reply README.md:20]: Honestly I'm hoping that they just haven't done much.

I think the easiest solution will be to have them make some patch files and then apply them to a fresh copy of the repo.  Like here: https://makandracards.com/makandra/2521-git-how-to-create-and-apply-patches

The other thing that makes it easy is if they have only done a single controller so far or something then they can just copy that one file over.

Is that enough?  I'm really just hoping that people that forked the wrong repo haven't started yet.

Comment (kaidamasaki) [reply README.md:20]: I don't hate the backup solution of just looking at both though.

Comment (kaidamasaki) [reply README.md:20]: Actually, I think I'm just going to update it to say we'll grade it but for them to let us know.

**Comment by jmaddox19** [comment]
Great, thanks for the quick update!

**Pull request merged**